homeassistant/components/auth/__init__.py

```diff
@@ -169,6 +170,14 @@
         vol.Required('refresh_token_id'): str,
     })
 
+WS_TYPE_SIGN_PATH = 'auth/sign_path'
+SCHEMA_WS_SIGN_PATH = \
+    websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend({
+        vol.Required('type'): WS_TYPE_SIGN_PATH,
+        vol.Required('path'): str,
```

Comment:
How about using `CONF_TYPE` and `CONF_PATH` from `const.py`?

Reply:
I don't know if we should conflate constants for configuration parameters and websocket parameters.

Reply:
Yeah, maybe better to keep them separated.